python/repositories.bzl

```diff
@@ -239,6 +239,7 @@ def _python_repository_impl(rctx):
             "libs/**",
             "Scripts/**",
             "share/**",
+            "tcl/**",
```

Comment:
Could you please add a `CHANGELOG.md` entry please?

Reply:
Done